whipper/common/mbngs.py

```diff
@@ -21,9 +21,10 @@
 """
 Handles communication with the MusicBrainz server using NGS.
 """
-
 import urllib2
 
+import whipper
+
 import logging
```

Comment:
I would probably do these lines like:
```Python
[…]
"""

import logging
import urllib2

import whipper

logger = …
```

(As per https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
It's standard to import logging on its own line right above where you define `logger`, so I'd like to keep it where it is.

Reply:
Right, but usually you first import system libs, then third party libs, then your own. I think that is what @Freso meant?

Reply:
It is! But I'm saying that despite pep8 it's common practice to import logging on its own line right above logger, out of standard order. As far as I can tell I'm otherwise following the standard import order. If someone feels really strongly about this I'm open to changing it, but this is the style whipper is currently written in and if we decide to change the import order for logging we should do it wholesale in another PR. (I'm fine discussing it here, however.)